std_msgs/Header header

float32[3] pos
float32[3] vel

float32[3] pos_sp
float32[3] vel_sp

float32[3] u_l
float32[3] u_d
float32[3] u_total

float32[3] thrust_sp



